modules/alpha/kustomization.yaml

```
@@ -1,3 +1,5 @@
 apiVersion: kustomize.config.k8s.io/v1alpha1
 kind: Component
 
+resources:
+- moduletemplate-keda-v0.0.1-alpha.yaml
```

```suggestion
- moduletemplate-keda.yaml
```
no need for the version

---

done